chromium_src/components/captive_portal/core/captive_portal_detector.cc

```diff
@@ -5,8 +5,21 @@
 
 #include "components/captive_portal/core/captive_portal_detector.h"
 
-#define kDefaultURL                                           \
-  kDefaultURL[] = "http://detectportal.brave-http-only.com/"; \
-  const char kEmpty
+namespace {
+
+constexpr char kBraveDefaultURL[] = "http://detectportal.brave-http-only.com/";
```

Comment:
this does not appear to be a valid URL so we should have a comment here explaining what we're doing with this override

Reply:
maybe this only works with the specific request made by the browser, I guess the first time I did it, I mistook it for not existing at all